sonar-ws/src/main/gen-java/org/sonarqube/ws/Rules.java

```diff
@@ -670,10 +613,6 @@ public Builder clear() {
           return this;
         }
 
-        public Builder clone() {
-          return create().mergeFrom(buildPartial());
-        }
-
         public com.google.protobuf.Descriptors.Descriptor
             getDescriptorForType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)